rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23.md

```diff
@@ -0,0 +1,41 @@
+# Proposal to upgrade the ledger suite orchestrator canister to add ckXAUT
+
+Git hash: `3d0b3f10417fc6708e8b5d844a0bac5e86f3e17d`
```

Comment:
This re-use the same version as before. The wasm upgrade due to the ic-cdk fix will be done in a separate proposal.